src/rf/apps/core/context_processors.py

```diff
@@ -23,11 +23,11 @@ def page_context(request):
 
 
 def get_client_settings():
-    conn = boto.connect_s3()
-    aws_key = conn.aws_access_key_id
+    creds = provider.Provider('aws')
     client_settings = {
         'signerUrl': reverse('sign_request'),
-        'awsKey': aws_key,
+        'awsKey': creds.get_access_key(),
```

Comment:
There is also `creds.access_key` and `creds.get_security_token()`. Might be good to stick with both as functions or both as accessors.